src/main/java/net/authorize/sample/SampleCode.java

```diff
@@ -18,7 +18,7 @@
  */
 public class SampleCode {
 
-    public static void main( String[] args )
+    public static void main( String[] args ) throws IllegalArgumentException, IllegalAccessException, NoSuchFieldException, SecurityException
```

Comment:
This change is unacceptable. It will break existing Travis CI flows

Reply:
Exceptions and throws removed form method definition.